akka-docs/src/main/paradox/stream/stream-refs.md

```diff
@@ -89,12 +89,11 @@ The process of preparing and running a `SourceRef` powered distributed stream is
 
 @@@ warning
   A `SourceRef` is *by design* "single-shot". i.e. it may only be materialized once.
-  This is in order to not complicate the mental model what materializing such value would mean.
+  This is in order to not complicate the mental model of what materialization means.
   
-  While stream refs are designed to be single shot, you may use them to mimic multicast scenarios,
-  simply by starting a `Broadcast` stage once, and attaching multiple new streams to it, for each
-  emitting a new stream ref. This way each output of the broadcast is by itself an unique single-shot
-  reference, however they can all be powered using a single `Source` -- located before the `Broadcast` stage.
+  Multicast can be mimicked by starting a `BroadcastHub` stage once then attaching multiple new streams to it, each
```

Comment:
I assume this meant to use BroadcastHub or was it a static broadcasting?

Reply:
It might have meant static ones, but good to change to Hub 👍